app/Actions/Diagnostics/Pipes/Checks/MigrationCheck.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Actions\Diagnostics\Pipes\Checks;
+
+use App\Contracts\DiagnosticPipe;
+use App\Metadata\Versions\FileVersion;
+use App\Metadata\Versions\LycheeVersion;
+
+class MigrationCheck implements DiagnosticPipe
+{
+	/**
+	 * @param array<int,string> $data list of error messages
+	 * @param \Closure(array<int,string> $data): array<int,string> $next
+	 *
+	 * @return array<int,string>
+	 */
+	public function handle(array &$data, \Closure $next): array
+	{
+		if (!self::isUpToDate()) {
+			$data[] = 'Error: Database is behind file versions. Please apply the migration.';
```

Comment:
```suggestion
			$data[] = 'Error: Database is behind file versions. Please apply the migrations.';
```
I know that it's only one migration that updates the version, but there are probably more migrations.